wpeverest/user-registration
# UR-556 Tweak- Add tag for selected my account page

### All Submissions:






### Changes proposed in this Pull Request:

Previously, There was only way to find out the selected my account page is from settings page. Now we have provide tag to Page that will help us know which page is selected my account.

### How to test the changes in this Pull Request:

1. Goto UR Settings -> select My Account Page.
2. Goto All Pages  where you can see **UR My Account Page** Tag in front of Page.

### Types of changes:




### Other information:




### Changelog entry

> Tweak- Add tag for selected my account page.
